### DISCUSSION TITLE
How do I find the source for a release version?

### DISCUSSION THREAD

**Discussion by corngood** [wieslawsoltes/Svg.Skia#241] [General]
I'm trying to package a source-build of `Avalonia.Svg.Skia`, but I can't quite figure out how to map versions as they appear on https://www.nuget.org/packages/Avalonia.Svg.Skia/ to commits in this repo (which is listed as the source repo).

It looks like 11.1.0 -> 2.0.0 and 11.1.0.1 -> 2.0.0.1, but that's just a hunch.

I suppose I can package it as version 2.0.0.1 even if it spits out nuget packages with version 11.1.01.  I just want to make sure I'm not missing something.

**Comment by wieslawsoltes** [comment]
![image](https://github.com/user-attachments/assets/19aea169-45c9-4774-89fd-a76d77239fc0)

**Reply by corngood** [reply]
Thanks.  So just to be clear, they are always published as a pair?  Like if 11.1.0.2 is there, there will be a corresponding 2.0.0.2?

I'm just asking so I know how to configure the auto-update mechanism in nixpkgs.  It sounds like we can track `Svg.Skia` and treat whatever version of `Avalonia.Svg.Skia` it spits out as a byproduct.
